scrapy_crawlera/middleware.py

```diff
@@ -278,14 +286,17 @@ def _get_slot(self, request):
         key = self._get_slot_key(request)
         return key, self.crawler.engine.downloader.slots.get(key)
 
-    def _set_custom_delay(self, request, delay):
+    def _set_custom_delay(self, request, delay, reason=None, update_stats=True):
```

Comment:
Why the `update_stats`? It seems it’s currently unused.

Reply:
Hi @Gallaecio , yes that's currently unused. I just wanted to make my changes relatively soft. :sweat_smile: 
Do you prefer it if I just remove this `update_stats` argument?

Reply:
I would prefer that. We can always add it back if we need it.

Reply:
Sure! PR updated now. Thanks for the advice. :)